packages/ds-ext/src/DiagramEditorProvider.ts

```diff
@@ -15,12 +15,13 @@ import { observeLinkUpdate } from './messageHandlers/observeLinkUpdate';
 import { getDataFromStorage } from './messageHandlers/getDataFromStorage';
 import { cancelObservation } from './messageHandlers/cancelObservation';
 import { DuckDBStorage } from './duckDBStorage';
+import { createDataStoryDBPath } from './commands/createDataStoryDBPath';
 
 export class DiagramEditorProvider implements vscode.CustomEditorProvider<DiagramDocument> {
   private readonly _onDidChangeCustomDocument = new vscode.EventEmitter<vscode.CustomDocumentEditEvent<DiagramDocument>>();
   public readonly onDidChangeCustomDocument = this._onDidChangeCustomDocument.event;
-  private inputObserverController: InputObserverController;
-  private duckDBStorage: DuckDBStorage;
+  private inputObserverController!: InputObserverController;
+  private duckDBStorage!: ObserverStorage;
```

Comment:
Consider changing the name `duckDBStorage` as it implies a exactly duck db implementation. Maybe `observerStorage` instead?